packages/shared/src/components/FeedItemComponent.tsx

```diff
@@ -12,10 +12,11 @@ import { LoggedUser } from '../lib/user';
 import { CommentOnData } from '../graphql/comments';
 import useTrackImpression from '../hooks/feed/useTrackImpression';
 import { FeedPostClick } from '../hooks/feed/useFeedOnPostClick';
-import { SharePostCard } from './cards/SharePostCard';
 import { WelcomePostCard } from './cards/WelcomePostCard';
 import { Origin } from '../lib/analytics';
 import { UseVotePost } from '../hooks';
+import CollectionCard from './cards/CollectionCard';
+import { SharePostCard } from './cards/SharePostCard';
```

Comment:
Not really important but see we export differently

Reply:
Good catch, it should be named import.